R/helpers.R

```diff
@@ -57,3 +57,27 @@ pprob_to_matrix <- function(pp, task) {
   colnames(y) <- task$class_names
   y
 }
+
+convert_ratio = function(pv, target, ratio, n) {
+  to_decimal = function(bits) {
```

Comment:
Does this mean `bits` in the computer sense? If not maybe rename?

Reply:
yes, bits as in computers. this function is now in mlr3misc.

Reply:
Okay so should we remove it here then?